### PR TITLE
Added v3.2.0 folder + docs for let block helper

### PR DESCRIPTION
**Pull request by kennethlarsen**
Reference: https://github.com/emberjs/ember.js/issues/16400

* Adds v3.2.0 to guides (Copy/paste from v3.1.0 - only real changes are in [built-in-helpers.hbs](https://github.com/kennethlarsen/guides-source/blob/096ba4131d9a6cdc51b0d537fbe74e2792737af2/guides/v3.2.0/templates/built-in-helpers.md) and [versions.yml](https://github.com/kennethlarsen/guides-source/blob/096ba4131d9a6cdc51b0d537fbe74e2792737af2/versions.yml))
* Adds documentation for the let helper

This is my first time writing guides docs so please let me know if changes are needed.

### CODE REVIEW

**Comment by kennethlarsen** [comment]
ping @locks

File: versions.yml [hunk 1]
```diff
@@ -24,4 +24,5 @@ allVersions:
   - "v2.18.0"
   - "v3.0.0"
   - "v3.1.0"
+  - "v3.2.0"
```
Comment (mansona) [inline versions.yml]: I wouldn't add anything to the versions.yml until we are ready to release the 3.2 docs

**Comment by sivakumar-kailasam** [comment]
@kennethlarsen I've added over 3.2.0 docs in a separate commit to master, if you can submit the changes from that it'd be easier to review.

**Comment by kennethlarsen** [comment]
Ahhh yes that makes totally sense. I'll close this PR and open a new, clean one.

**Pull request closed without merging**